components/Footer.tsx

```diff
@@ -16,27 +16,21 @@ const Footer = () => {
         <SocialColumn>
           <LogoGraphic src={Logo} />
           <SocialLinks>
-            <Link href="mailto:hackers@hacksc.com">
+            <a href="mailto:hackers@hacksc.com">
               <img src={Email} />
-            </Link>
-            <Link
-              href="https://www.facebook.com/hackscofficial/"
-              target="_blank"
-            >
+            </a>
+            <a href="https://www.facebook.com/hackscofficial/" target="_blank">
               <img src={Facebook} />
-            </Link>
-            <Link
-              href="https://www.instagram.com/hackscofficial/"
-              target="_blank"
-            >
+            </a>
+            <a href="https://www.instagram.com/hackscofficial/" target="_blank">
               <img src={Instagram} />
-            </Link>
-            <Link href="https://medium.com/@hacksc" target="_blank">
+            </a>
+            <a href="https://medium.com/@hacksc" target="_blank">
```

Comment:
Whats the reasoning to moving everything from Links?

Reply:
kept getting console errors so went ahead and fixed